zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, encoding='utf-8', mode='w') as f:
```

Comment:
Thanks for the feedback, I was not aware that this is available only from python 3. Anyway, I tested it with a small python script:
```
import urllib2
response = urllib2.urlopen('http://zap:8090/OTHER/core/other/htmlreport/formMethod=GET')
report = response.read()
with open('/usr/src/wrk/zap.html', 'wb') as f:
    f.write(report)
```
And it worked perfectly (python 2.7). Also put the report in a [gist](https://gist.github.com/omerlh/a974942028e744760374b62bca57fc0e), in case you want to take a look.